index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <title>Ed's Homepage</title>
+  <link rel="stylesheet" href="css/homepage.css" type="text/css" />
+    <link rel="stylesheet" href="css/variables.css" type="text/css" />
+</head>
+<body>
+  <div class="wrap">
+    <div class="title">
+        <h1> Ed Freeman </h1>
+    </div>
+    <div class="content">
+      <div class="col col-1">
+  			<h2>CV</h>
```

Comment:
I can kind of see why you did it, but this indenting is pretty weird

Reply:
Interestingly... it displays as one tab for me, but here it displays as like 3.... Looking into.

Reply:
I am personally a proponent of spaces > tabs, but that's a legitimate argument had by many devs

Reply:
Oh really?  Hmmm

Reply:
Pretty sure I've fixed: https://github.com/Zenloth/zenloth.github.io/commit/46b25edfe5d74f9ce09b949d1e5718ded8edef0b